Documentation/op-guide/failures.md

```diff
@@ -30,7 +30,7 @@ Once a majority of members works, the etcd cluster elects a new leader automatic
 
 ## Network partition
 
-A network partition is similar to a minor followers failure or a leader failure. A network partition divides the etcd cluster into two parts; one with a member majority and the other with a member minority. The majority side becomes the available cluster and the minority side is unavailable; there is no “split-brain” in etcd. 
+A network partition is similar to a minor followers failure or a leader failure. A network partition divides the etcd cluster into two parts; one with a member majority and the other with a member minority. The majority side becomes the available cluster and the minority side is unavailable; there is no “split-brain” in etcd.  ( no “split-brain” is possible because etcd cluster members are explicitly added/removed, with each such change approved by the current majority, and thus allows cluster members can determine majority. ) 
```

Comment:
I am good with added detail but nit if you and @ptabor agree, I would remove brackets and preferably re-write something like `The majority side becomes the available cluster and the minority side is unavailable. There is no “split-brain” in etcd because etcd cluster members are explicitly added/removed with each such change is approved by the current majority of members.`  Thanks!

Reply:
Thanks again.  Agreed that is looks better without braces.

Reply:
@randomizedcoder thank you for addressing comment quickly. I see 3 commits so would you please squash them, and then we merge it. Thanks!

Reply:
Thanks again.  Hmmm.  Sorry, can I squash them?  I don't see it here in the web GUI.  I normally do the squash via squash merge, which I think is something only users with merge can do.  I guess I could submit a new PR with the single commit.